.github/workflows/tox.yml

```diff
@@ -1,16 +1,24 @@
 name: tox
-on: [push, pull_request]
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
 jobs:
   tox:
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        python: ['3.6', '3.7', '3.8', '3.9', '3.10']
+        python: ['3.6', '3.7', '3.8', '3.9', '3.10', '3.11']
```

Comment:
It looks like GitHub doesn't have a python 3.6 runner available for ubuntu-latest which is currently 22.04; I think we'd need to either drop 3.6 or pin ubuntu to 20.04.

Reply:
Correct.  Which way should we go?
* https://devguide.python.org/versions

Reply:
My vote would be to retain support for 3.6 and pin ubuntu to 20.04 for now. That seems to me like it'd have the least negative impact right now for consumers of this library.